addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -1251,7 +1251,7 @@ const SnippetOptionWidget = Widget.extend({
             widgetValue = `var(--${widgetValue})`;
         }
 
-        const values = widgetValue.split(/\s+/g);
+        const values = widgetValue.replace(/,\s/g, ',').split(/\s+/g);
```

Comment:
@ThanhDodeurOdoo All of this did not exist in 13.0 so 13.1 is the right version. However, maybe add a comment explaining why we do that replace.